brave/brave6/src/main/java/com/linecorp/armeria/server/brave/ArmeriaRpcServerParser.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2025 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.brave;
+
+import brave.rpc.RpcRequestParser;
+import brave.rpc.RpcResponseParser;
+
+/**
+ * Provides Armeria's default implementation for server-side RPC request and response parsers.
+ * Users may use the default parser like the following:
+ * <pre>{@code
+ * Tracing tracing = ...
+ * RpcTracing rpcTracing =
+ *   RpcTracing.newBuilder(tracing)
+ *              .serverRequestParser((req, ctx, span) -> {
+ *                  // apply brave's default request parser
+ *                  RpcRequestParser.DEFAULT.parse(req, ctx, span);
+ *                  // apply armeria's default request parser
+ *                  ArmeriaRpcServerParser.requestParser().parse(req, ctx, span);
+ *              })
+ *              .serverResponseParser((res, ctx, span) -> {
+ *                  // apply brave's default response parser
+ *                  RpcResponseParser.DEFAULT.parse(res, ctx, span);
+ *                  // apply armeria's default response parser
+ *                  ArmeriaRpcServerParser.responseParser().parse(res, ctx, span);
+ *              });
+ * BraveRpcService
+ *   .newDecorator(rpcTracing)
+ *   ...
+ * }</pre>
+ * The following tags will be available by default:
+ * <ul>
+ *   <li>http.url</li>
+ *   <li>http.host</li>
+ *   <li>http.protocol</li>
+ *   <li>http.serfmt</li>
+ *   <li>address.remote</li>
+ *   <li>address.local</li>
+ * </ul>
+ */
+public final class ArmeriaRpcServerParser {
```

Comment:
- `@UnstableApi`
- How about merging this class into `ArmeriaHttpServerParser` and provide all factory methods there? e.g. `BraveServerParsers.rpcRequestParser()`

Reply:
> How about merging this class into ArmeriaHttpServerParser and provide all factory methods there?

I was assuming that most users will be interested in `Http` related integrations only.
Do you prefer that `libs.brave6.instrumentation.rpc` is included with the `api` scope instead?

Reply:
> How about merging this class into ArmeriaHttpServerParser and provide all factory methods there? e.g. BraveServerParsers.rpcRequestParser()

Just merged the class and added the dependency since it seems you prefer this way.